src/mono/wasm/Makefile

```diff
@@ -117,7 +117,7 @@ $(BUILDS_OBJ_DIR)/corebindings.o: runtime/corebindings.c | $(BUILDS_OBJ_DIR)
 	$(EMCC) $(EMCC_FLAGS) $(1) -Oz -I$(MONO_INCLUDE_DIR) runtime/corebindings.c -c -o $$@
 
 $(NATIVE_BIN_DIR)/src/emcc-flags.txt: | $(NATIVE_BIN_DIR)/src Makefile
-	echo "$(call escape_quote,$(EMCC_FLAGS)) $(1) -Oz" > $$@
+	echo "$(call escape_quote,$(EMCC_FLAGS)) $(1)" > $$@
```

Comment:
Can we add this, and the other changes to the PR description? Maybe like:

```
- Always set `$(WasmNativeStrip)` to `false`, for library tests
- Remove `-Oz` from default emcc flags
- New `$(_WasmDevel)` to build native files with `-O0`
```

Or corrected, if I have some of that wrong!

Reply:
They are in the commit descriptions, will be in the final merged commit message.

Reply:
Sorry, I missed all those!